geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionConfigImpl.java

```diff
@@ -2761,13 +2763,19 @@ public void setServerSSLAlias(final String alias) {
   }
 
   @Override
-  public boolean getSSLEndpointIdentificationEnabled() {
-    return sslEndpointIdentificationEnabled;
+  public Boolean getSSLEndPointIdentificationEnabled() {
```

Comment:
since this would never return null, I would have it return "boolean" instead of "Boolean"